tasks/src/test/java/taf/WebDriverYandexDiskTest.java

```diff
@@ -0,0 +1,65 @@
+package taf;
+
+import static taf.constants.Constants.*;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import taf.service.YandexDiskService;
+
+public class WebDriverYandexDiskTest {
+  private WebDriver driver;
+
+  @BeforeMethod(alwaysRun = true,
+      description = "Google Chrome opens, goes to Yandex Disk Auth page and logged in with correct credentials")
+  public void goToDiskMainPage() {
```

Comment:
Duplicated code snippet. I suggest you move @Before and @After methods that you wanna reuse into a base test for example.